BitwardenShared/UI/Auth/VaultUnlock/VaultUnlockState.swift

```diff
@@ -29,6 +29,9 @@ struct VaultUnlockState: Equatable {
     /// A flag indicating if the master password should be revealed or not.
     var isMasterPasswordRevealed = false
 
+    /// A flag indicating if the password/pin text field should be auto focused.
+    var isPasswordAutoFocused = true
+
```

Comment:
⛏️ Do we actually need this in the state? I mean unless I'm missing something it will always be `true`, so I guess we could just set the `true` value directly in the View. Maybe I'm breaking some pattern here, if so let me know. cc: @matt-livefront @KatherineInCode

Reply:
yeah, i went back and forth on this myself... I opted to follow the pattern but in this case it might not be worth it?  im good with whatever the team thinks is best here.

Reply:
If possible, I think we should use `@FocusState` for this. Ideally we would have focus in the state, but SwiftUI makes it difficult (`@FocusState` specifically). I think it can just be in the view if we don't have a reason to change it from the processor. But if we did need a static bool like this, I think keeping it in the view is ok.